openqaoa/backends/qpus/qaoa_qiskit_qpu.py

```diff
@@ -183,6 +183,7 @@ def get_counts(self, params: QAOAVariationalBaseParams) -> dict:
 
             while api_contact == False:
                 try:
+                    self.job_id = job.job_id()
```

Comment:
Do we save the jobids for the other backends too?

Reply:
@shahidee44 , maybe you can answer this one?